CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.6...4.1)
```

Comment:
I looked up cmake 2.6. It does support the `POST_BUILD` key word below, so we can keep the minimum at that: https://cmake.org/cmake/help/cmake2.6docs.html#command:add_custom_command